### PR TITLE
Append GroupVersion if new

### LINKED ISSUE #9: Bug: Kind should be provided from ***Hook interface
Hi @deads2k , I noticed that Kind is hardcoded to `AdmissionReview` in https://github.com/openshift/generic-admission-server/blob/master/pkg/apiserver/apiserver.go#L174

Should this be provided from the **Hook interface, since users of this library can define the singular and plural form the resource.

### PR DESCRIPTION
**Pull request by tamalsaha**
Fixes #9

### CODE REVIEW

**Comment by tamalsaha** [comment]
cc: @deads2k

File: pkg/apiserver/apiserver.go [hunk 1]
```diff
@@ -210,6 +210,21 @@ func (c completedConfig) New() (*AdmissionServer, error) {
 	return s, nil
 }
 
+func appendGroupVersion(slice []schema.GroupVersion, elems ...schema.GroupVersion) []schema.GroupVersion {
```
Comment (deads2k) [inline pkg/apiserver/apiserver.go]: nit: appendUniqueGroupVersion?

**Comment by deads2k** [comment]
A nit if you get to it in the next few hours.  Otherwise, I'll merge as-is.  Good catch.

**Comment by tamalsaha** [comment]
Updated. PTAL.

**Comment by deads2k** [comment]
lgtm

**Pull request merged**